spec/serializers/user_bookmark_serializer_spec.rb

```diff
@@ -33,6 +33,20 @@
     expect(s.post_user_avatar_template).not_to eq(nil)
   end
 
+  it "uses the correct highest_post_number column based on whether the user is staff" do
+    Fabricate(:post, topic: bookmark.topic)
+    Fabricate(:post, topic: bookmark.topic)
+    Fabricate(:whisper, topic: bookmark.topic)
+    list = BookmarkQuery.new(user: bookmark.user).list_all.to_ary
+    serializer = UserBookmarkSerializer.new(list.last, scope: Guardian.new(user))
```

Comment:
Since this is a unit test for the serializer, I think we can just pass the bookmark directly to the serializer instead of having to query for it.